screens/Scan/SuccessfullySharedModal.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import {useTranslation} from 'react-i18next';
 import {Theme} from '../../components/ui/styleUtils';
 import {Modal} from '../../components/ui/Modal';
-import {Image} from 'react-native';
-import {Column, Text} from '../../components/ui';
-import {Button} from '../../components/ui';
+import {Pressable} from 'react-native';
```

Comment:
Can we combine these import to avoid duplicate ?
`import {Dimensions, Pressable} from 'react-native';`

Reply:
Done